src/Tester/GenericGrainTests.cs

```diff
@@ -32,7 +32,7 @@ MIT License
 using UnitTests.Tester;
 using System.Collections.Generic;
 
-namespace UnitTests.General
```

Comment:
all our namespaces in this project are: `namespace UnitTests.General`. Don't see a reason to change everything now.